torch/onnx/symbolic_helper.py

```diff
@@ -963,45 +963,15 @@ def requantize_bias_helper(g, bias, input_scale, weight_scale):
                   to_i=torch.onnx.TensorProtoDataType.INT32)
     return g.op("prim::TupleConstruct", q_bias, bias_scale, bias_zero_point)
 
-# ---------------------------------------------------------------------
```

Comment:
Isnt this valuable anymore?

Reply:
I don't think so. I think it may be a relic of some older version of the code where the symbolic functions were not separated into per-`opset_version` files, and the same functions would be called regardless of the value of this variable.

In any case the comment doesn't match my understanding of the current reality, because the way the symbolic functions are called now respects whatever `opset_version` is being used, and it doesn't really matter what the default is.

LMK if you think there's something from this comment that's still true and worth keeping here..